repos/spack_repo/builtin/packages/mathic/package.py

```diff
@@ -17,11 +17,13 @@ class Mathic(AutotoolsPackage):
 
     homepage = "https://github.com/Macaulay2/mathic"
     git = "https://github.com/Macaulay2/mathic"
+    url = "https://github.com/Macaulay2/mathic/releases/download/v1.1/mathic-1.1.tar.gz"
```

Comment:
```suggestion
homepage = "https://github.com/Macaulay2/mathic"
url = "https://github.com/Macaulay2/mathic/releases/download/v1.1/mathic-1.1.tar.gz"
git = "https://github.com/Macaulay2/mathic"
```

Let's swap `url` and `git` for consistency across packages.

Reply:
Done!